ntoskrnl/io/pnpmgr/devnode.c

```diff
@@ -84,7 +129,7 @@ PipAllocateDeviceNode(
  *
  * @return     Status, indicating the result of an operation
  */
-
+#if 0
```

Comment:
Is that the thing about the legacy drivers that won't completely be supported now?

Reply:
Yeah, I left it because of that. But tbh, I'm not sure if those entries are enough for legacy devices to work properly